tox.ini

```diff
@@ -82,7 +82,7 @@ commands = mypy aea packages tests scripts
 [testenv:bandit-main]
 basepython = python3.7
 deps = bandit
-commands = bandit -s B101,B105,B301,B403,B404,B603,B605,B607,B608 -r aea packages scripts
```

Comment:
@marcofavorito B605 and B608 should also be removed

Reply:
Why are you removing `packages` from here?

Reply:
shouldn't it include `packages/`?